x-pack/legacy/plugins/beats_management/server/rest_api/beats/enroll.ts

```diff
@@ -34,7 +33,7 @@ export const createBeatEnrollmentRoute = (libs: CMServerLibs) => ({
   },
   handler: async (
     request: FrameworkRequest
-  ): Promise<BaseReturnType | ReturnTypeCreate<CMBeat>> => {
+  ): Promise<BaseReturnType | ReturnTypeCreate<string>> => {
```

Comment:
@mattapperson this looks like the correct return type right?

Reply:
Negative, CMBeat is an object defined here https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/beats_management/common/domain_types.ts#L41